source/algorithms.tex

```diff
@@ -3998,7 +3998,8 @@
 such that for every non-negative integer \tcode{n} less than \tcode{count}
 the following corresponding conditions hold:
 \tcode{*(i + n) == value, pred(*(i + n),value) != false}.
-Returns \tcode{last} if no such iterator is found.
+Returns \tcode{first} if \tcode{count <= 0},
```

Comment:
This normatively duplicates the stated behavior of the first sentence. If we want to clarify the meaning of the first sentence for this case, then this should be a note.

Reply:
And why is this extra provision present for `search` (without `_n`)? Given the similar structure, I'd expect the "normative duplication" would applie to both.

Reply:
I consider it a defect that the redundant provision is present for `std::search`. Notably none of `std::ranges::search`, `std::search_n`, nor `std::ranges::search_n` have this defect. Spreading the provision from one member of this set to a second does not harmonize the wording of all four.

Reply:
Thanks. So, we should have an LWG issue to remove the duplication from `search` and maybe add a note everywhere talking about the "empty range" case. Unless @zygoloid this approach would be editorial.